### PR TITLE
Return error through Rx to make the error handling easier

### PR DESCRIPTION
**Pull request by lucas34**
PR for issue #29

### CODE REVIEW

**Comment by pwittchen** [comment]
Thanks for your PR. I'll test it and when everything will be fine, I'm gonna merge it soon.

File: library/src/main/java/com/github/pwittchen/reactivesensors/library/SensorNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.github.pwittchen.reactivesensors.library;
+
+/**
+ * Created by lucas34990 on 5/12/16.
+ */
```
Comment (pwittchen) [inline library/src/main/java/com/github/pwittchen/reactivesensors/library/SensorNotFoundException.java]: I generally don't put "created by" comments to my projects. This information is accessible through Git.

File: library/src/main/java/com/github/pwittchen/reactivesensors/library/SensorNotFoundException.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+public class SensorNotFoundException extends Exception {
+
+    public SensorNotFoundException(String detailMessage) {
```
Comment (pwittchen) [inline library/src/main/java/com/github/pwittchen/reactivesensors/library/SensorNotFoundException.java]: Variable named `message` would be enough. Moreover, this parameter can be `final`.

**Comment by lucas34** [comment]
No worries

**Comment by pwittchen** [comment]
PR merged. Thanks!

**Pull request merged**